google/cloud/speech/cloud_speech.yaml

```diff
@@ -12,18 +12,8 @@ documentation:
 
 apis:
 - name: google.cloud.speech.v1beta1.Speech
-# TODO get LRO API call support added for Speech
-#- name: google.longrunning.Operations
+- name: google.longrunning.Operations
```

Comment:
To confirm, is the intended usage that a user must explicitly install/import a separate LRO client in order to query operations if they are using, for example, Cloud Speech?